gax/src/main/java/com/google/api/gax/batching/v2/BatcherImpl.java

```diff
@@ -72,11 +73,14 @@
   public BatcherImpl(
```

Comment:
(It was probably missed during previous reviews). It is a Public constructor of public class. Please add java doc for it.

Reply:
Does it need to actually be public?

Reply:
Yes, it will be instantiated but the stub. Ie:

‘’’java
class BigtableStub {
...
Batcher<RowMutation,void> newMutateRowsBatcher(String tableId) { return BatchetImpl.NewBuilder()....build();}

Reply:
Have added Javadoc on BatchImpl constructor. Please let me know if this looks good to you.

Reply:
Also, please mark this `@InternalApi("For google-cloud-java client use only")`